### DISCUSSION TITLE
Manually add missing charge session

### DISCUSSION THREAD

**Discussion by cleanev** [teslamate-org/teslamate#1887] [Q&A]
I have a missing charge session that I would like to add manually, however I am not very well versed in sql scripting so would like some help if it is possible to add this session manually. It so happens that I can check my tesla account and get details of kWh and price to include in this addition.
From what I recall, the date and time coincides with our internet down for a few hours on that day and therefore this seems to be missed.
Any help will be appreciated.
Thanks in advance.

**Comment by ghost** [comment]
I have the same question.
Is it possible to install something like phpMyAdmin?
